securedrop_proxy/proxy.py

```diff
@@ -154,6 +152,24 @@ def simple_error(self, status: int, err: str) -> None:
 
         self.res = res
 
+    def normalize_path(self, parsed: ParseResult) -> ParseResult:
+        """
+        This is copied from the furl library (Unlicense)
```

Comment:
I think we should explicitly use `SPDX-License-Identifier:`. At least, that would make it easier in the future to find instances where subsections of our codebase are mixing in code licensed differently. (I used it https://github.com/freedomofpress/securedrop/search?q=spdx here)

Reply:
Done!